src/main/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEvent.java

```diff
@@ -212,41 +211,41 @@ private CaseData getCaseData(final CaseData caseData, final DssCaseData dssCaseD
         }
 
         List<CaseworkerCICDocument> docList = new ArrayList<>();
-        List<ListValue<DssMessage>> listValues = new ArrayList<>();
+        List<ListValue<DssMessage>> messagesList = new ArrayList<>();
 
         if (isNotEmpty(dssCaseData.getOtherInfoDocuments())) {
-            //For showing additional information page data on tab
             for (ListValue<EdgeCaseDocument> documentListValue : dssCaseData.getOtherInfoDocuments()) {
                 Document doc = documentListValue.getValue().getDocumentLink();
+                String documentComment = documentListValue.getValue().getComment();
                 doc.setCategoryId(DocumentType.DSS_OTHER.getCategory());
                 CaseworkerCICDocument caseworkerCICDocument = CaseworkerCICDocument.builder()
                     .documentLink(doc)
                     .documentCategory(DocumentType.DSS_OTHER)
+                    .documentEmailContent(documentComment)
                     .build();
 
                 if (!docList.contains(caseworkerCICDocument)) {
                     docList.add(caseworkerCICDocument);
                 }
+            }
+        }
 
-                final User caseworkerUser = idamService.retrieveUser(request.getHeader(AUTHORIZATION));
-
-                final DssMessage message = DssMessage.builder()
-                    .message(dssCaseData.getAdditionalInformation())
-                    .dateReceived(LocalDate.now())
-                    .receivedFrom(caseworkerUser.getUserDetails().getFullName())
-                    .documentRelevance(dssCaseData.getDocumentRelevance())
-                    .build();
-
-                final ListValue<DssMessage> listValue = ListValue
-                    .<DssMessage>builder()
-                    .id(UUID.randomUUID().toString())
-                    .value(message)
-                    .build();
+        if (isNotBlank(dssCaseData.getAdditionalInformation())) {
+            final User caseworkerUser = idamService.retrieveUser(request.getHeader(AUTHORIZATION));
+            final DssMessage message = DssMessage.builder()
+                .message(dssCaseData.getAdditionalInformation())
+                .dateReceived(LocalDate.now())
+                .receivedFrom(caseworkerUser.getUserDetails().getFullName())
+                .build();
 
-                listValues.add(listValue);
-            }
+            final ListValue<DssMessage> listValue = ListValue
```

Comment:
why not move the creation of messagesList down here as well since it is only needed in this block?